### ISSUE TITLE
Label Android client's sliders

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#2516]
### Description

Currently, everything such as buttons, switches has lables, but only seeck bars doesn't have lables. those cannot be labled with android:text, so I'll use content discription to lable those for screen readers, because android says that because seeckbars are dinamic, if we want to lable them for screen readers, we should use content discription for them. I'll soon work on these issues. I opened them so that I can remember.

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by bear101** [comment]
Do you plan to fix this? Otherwise I'll move it to Discussions' feature requests.
